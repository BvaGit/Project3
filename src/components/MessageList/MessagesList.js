import React from "react";

import MessageInput from "./MessageInput";
import MessageItem from "./MessageItem";

const MessagesList = ({ messages }) => {
  return (
    <>
      <MessageItem messages={messages} />
      <MessageInput />
    </>
  );
};

export default MessagesList;
