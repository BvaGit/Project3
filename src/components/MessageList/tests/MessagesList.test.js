import React from 'react';
import { shallow, mount } from 'enzyme';
import MessagesList from '../MessagesList';


describe('MessagesList', () => {
    it("Should match snapshot", () => {
        const component = shallow(<MessagesList />)
        expect(component).toMatchSnapshot()
    })
})