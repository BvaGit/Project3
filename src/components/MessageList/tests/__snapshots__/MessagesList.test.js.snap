// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessagesList Should match snapshot 1`] = `ShallowWrapper {}`;
